client/components/header-cake/index.jsx

```diff
@@ -47,6 +52,10 @@ HeaderCake.propTypes = {
 	onTitleClick: PropTypes.func,
 	backText: PropTypes.string,
 	backHref: PropTypes.string,
+	actionText: PropTypes.string,
+	actionHref: PropTypes.string,
+	actionIcon: PropTypes.string,
+	actionOnClick: PropTypes.func,
```

Comment:
Absolutely!  I was waiting to make sure the solution was acceptable prior to updating the README ;)